snow/engine/snowman/voter.go

```diff
@@ -67,15 +89,25 @@ func (v *voter) Update() {
 
 func (v *voter) bubbleVotes(votes ids.Bag) ids.Bag {
 	bubbledVotes := ids.Bag{}
+	var err error
 	for _, vote := range votes.List() {
 		count := votes.Count(vote)
-		blk, err := v.t.VM.GetBlock(vote)
-		if err != nil {
-			continue
+		blk, ok := v.t.processing[vote.Key()] // Check if the block is non-dropped and processing
+		if !ok {
+			blkIntf, ok := v.t.droppedCache.Get(vote) // See if the block was dropped
```

Comment:
We should put a comment here that we are assuming that blocks that require votes to be bubbled are sufficiently small to fit into the dropped cache.